import-validator-accounts.yaml

```diff
@@ -9,16 +9,15 @@
       include_role:
         name: stop-services
 
-    - name: Set permissions of import folder
-      shell: "chown -R {{ stereum_user }}:{{ stereum_user }} {{ e2dc_install_path }}/launchpad"
-      when: setup != "multiclient"
-
     - name: Copy lanchpad wallet's validator keys
       shell: "cp {{ validator_keys_path }}/keystore-*.json {{ e2dc_install_path }}/launchpad"
       become: yes
-      become_user: "{{ stereum_user }}"
       when: setup != "multiclient"
 
+    - name: Set permissions of import folder
```

Comment:
is there no `become: yes` necessary?

Reply:
I tested it without `become: yes` and so far it works. But I think you it would be more sure